dbfit-java/core/src/main/java/dbfit/api/vendor/DbStoredProcedureCall.java

```diff
@@ -0,0 +1,16 @@
+package dbfit.api.vendor;
+
+import java.sql.SQLException;
+
+import dbfit.fixture.StatementExecution;
+
+public interface DbStoredProcedureCall {
+
+    String getName();
+
+    boolean isFunction();
+
+    String toSqlString();
```

Comment:
Stripping out both `toSqlString()` and `isFunction` we're left with just: -

```
package dbfit.api.vendor;

import java.sql.SQLException;

import dbfit.fixture.StatementExecution;

public interface DbStoredProcedureCall {

    StatementExecution toStatementExecution() throws SQLException;
}
```

On `DbObject` we already have `StatementExecution buildPreparedStatement(DbParameterAccessor accessors[])` and for stored routines this creates the appropriate procedure/function executor but for tables creates an `INSERT` (not other types) statement executor. This is a bit obscure.

Perhaps we don't need a separate `DbStoredProcedureCall` interface and implementations. Perhaps the classes implementing `DbObject` should provide these executors directly. What do you think?